app/views/alchemy/admin/attachments/_filter_bar.html.erb

```diff
@@ -0,0 +1,32 @@
+<% params_to_keep = [:tagged_with, :q, :content_id, :options] %>
+
+<div id="filter_bar">
+  <h2><%= Alchemy.t('Filter') %></h2>
```

Comment:
Or have  a heading for the entire filter bar, and then subheadings for "File type" and "Tag".